### PR TITLE
ENH: Update to support new .srep.json files

### PR DESCRIPTION
**Pull request by Connor-Bowley**

### CODE REVIEW

**Comment by bpaniagua** [comment]
@yehan0223 can you please review?

File: src/ShapePopulationData.cxx [hunk 1]
```diff
@@ -1,5 +1,41 @@
 #include "ShapePopulationData.h"
 
+#include <qSlicerCoreApplication.h>
+#include <vtkMRMLScene.h>
+#include <vtkMRMLSRepNode.h>
+#include <vtkMRMLSRepStorageNode.h>
+#include <vtkSlicerApplicationLogic.h>
+#include <vtkSlicerSRepLogic.h>
+#include <vtkSRepExportPolyDataProperties.h>
```
Comment (yehan0223) [inline src/ShapePopulationData.cxx:9]: @Connor-Bowley I am building the SPV PR branch using the current SlicerSALT master, the build couldn't find vtkSRepExportPolyDataProperties.h, could you check where this file is?

Comment (yehan0223) [reply src/ShapePopulationData.cxx:9]: I see that this file is in your SRep2 local branch under PR reviewing. We probably want to merge it to the SRep2 master, update SlicerSALT master, and then I will check from there.

Comment (Connor-Bowley) [reply src/ShapePopulationData.cxx:9]: Sounds good.

Comment (Connor-Bowley) [reply src/ShapePopulationData.cxx:9]: @yehan0223 Here is the PR bringing the SRep2 changes into SlicerSALT master.
https://github.com/Kitware/SlicerSALT/pull/253

**Comment by yehan0223** [comment]
@bpaniagua I tested this by pulling the SPV PR branch from SlicerSALT superbuild since there is compiling issue with SPV itself. It works great for both .xml and .srep.json files. Please go ahead with merging it if you are ok with the visuals of the medial sheets.

@Connor-Bowley you may lift the "Draft" now and update SlicerSALT superbuild after this PR is merged. 

Screenshot from 2022-03-30 17-04-59

**Comment by bpaniagua** [comment]
lgtm, feel free to merge

**Pull request merged**